modules/distribution/src/repository/jaggeryapps/cdm/controller/acs.jag

```diff
@@ -24,16 +24,20 @@ var keyStoreParams = {
     keyStoreName: dataConfig.ssoConfiguration.keyStoreName,
     keyStorePassword: dataConfig.ssoConfiguration.keyStorePassword,
     identityAlias: dataConfig.ssoConfiguration.identityAlias
-}
+};
 sso.configure(dataConfig.ssoConfiguration.issuer, dataConfig.ssoConfiguration.appName, keyStoreParams,
 dataConfig.ssoConfiguration.identityProviderURL);
 sso.acs(function(loggedInUser) {
     session.put(constants.USER_SESSION_KEY, loggedInUser);
     var username = loggedInUser;
-    log.debug("User logged in: "+username);
+    if(log.isDebugEnabled()){
+        log.debug("User logged in: "+username);
```

Comment:
Formatting issues. Please refer the segment >> log.debug("User logged in: "+username);.. Proper spacing need to be done between tokens like "+", "username", etc.